test/Parse/ConditionalCompilation/endianness.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-parse-verify-swift
+// RUN: %target-swift-frontend -emit-sil %s | FileCheck %s
+
+// CHECK:      [[X:%[0-9]+]] = integer_literal $Builtin.Int1, -1
+// CHECK-NEXT: {{%[0-9]+}} = struct $Bool ([[X]] : $Builtin.Int1)
+
+#if arch(arm) || arch(arm64) || arch(i386) || arch(x86_64) || arch(powerpc64le)
```

Comment:
I'd recommend adding the endianness checks to each file in `test/Parse/ConditionalCompilation` instead (just after the `_runtime()` clause).  This ensures that the tests run always, and not just when targeting that platform.